naga/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "naga"
-version = "0.20.0"
+version = "0.20.1"
```

Comment:
Naga didn't have any changes, so doesn't need to be bumped

Reply:
I think I've resolved this. Double-check plz? 🥺